common/ECSResourcesCheck/src/main/java/Comparer.java

```diff
@@ -0,0 +1,70 @@
+import org.json.JSONObject;
+public class Comparer {
+
+    public void compare(JSONObject jsonBase, String cluster,ECSResources ecs) {
+        Object[] arr = jsonBase.keySet().toArray();
+        JSONObject output = new JSONObject();
+        int key=jsonBase.keySet().toArray().length;
+        for (int i = 0; i < key; i++) {
+            System.out.println(arr[i].toString());
```

Comment:
arr[i].toString() is used in 3 places. Put its value in a variable and refers to the variable instead.

Reply:
Replaced with variable "module"